runexporterapp.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
Let's merge the contents of this file into runironic-exporter.sh to avoid confusion?

Reply:
sure, merged